File: java_projects/glue_code/commons-validator/src/test/java/org/apache/commons/validator/ResultPair.java
package org.apache.commons.validator;

import org.graalvm.polyglot.Value;

public class ResultPair {
  private static Value clz = ContextInitializer.getPythonClass("/ResultPair.py", "ResultPair");
  private Value obj;

  public ResultPair(Value obj) {
    this.obj = obj;
  }

  public Value getPythonObject() {
    return obj;
  }

  public ResultPair(String item, boolean valid) {
    //
    // this.item = item;
    // this.valid = valid; // Whether the individual part of url is valid.
    //

    this.obj = clz.invokeMember("__init__", item, valid);
  }
}

File: data/recomposed_projects/commons-validator/src/test/org/apache/commons/validator/ResultPair.py
# Imports Begin
import unittest

# Imports End


class ResultPair(unittest.TestCase):

    # Class Fields Begin
    item: str = None
    valid: bool = None
    # Class Fields End

    # Class Methods Begin
    def __init__(self, item: str, valid: bool) -> None:

        self.item = item
        self.valid = valid

    # Class Methods End
